packages/core/src/components/badge/badge.tsx

```diff
@@ -0,0 +1,20 @@
+import { Component, h, Prop } from '@stencil/core';
+import clsx from 'clsx';
+
+@Component({
+  tag: 'dso-badge',
+  styleUrl: 'badge.scss',
+  shadow: true
+})
+export class Badge {
+  @Prop()
+  level: 'primary' | 'success' | 'info' | 'warning' | 'danger';
```

Comment:
* Ik twijfel over de naamgeving `level`, kan je uitzoeken hoe het bij andere frameworks heet?
* `level` is in optioneel, als er geen `level` is wordt er ook geen `badge-` modifier gezet. Dan krijgen we een grijze badge

Reply:
Zie https://github.com/dso-toolkit/dso-toolkit/pull/865/files#diff-e457271040083984e3206598c3cf24b1082688a0c79d63a9c1a25b3b93e6dbe4R2